pkg/puppet/puppet.go

```diff
@@ -170,6 +170,9 @@ func kubernetesClusterConfigPerRole(conf *clusterv1alpha1.ClusterKubernetes, rol
 		if conf.ClusterAutoscaler.Version != "" {
 			hieraData.variables = append(hieraData.variables, fmt.Sprintf(`kubernetes_addons::cluster_autoscaler::version: "%s"`, conf.ClusterAutoscaler.Version))
 		}
+		if conf.ClusterAutoscaler.ScaleDownThreshold != 0 {
```

Comment:
we should check here that the threshold is not less than 0 or greater than 1, or put something in https://github.com/jetstack/tarmak/blob/master/pkg/tarmak/cluster/cluster.go to error when that isn't the case